pkg/scheduler/api/node_info.go

```diff
@@ -356,11 +356,11 @@ func (ni *NodeInfo) setNodeOthersResource(node *v1.Node) {
 // setNode sets kubernetes node object to nodeInfo object without assertion
 func (ni *NodeInfo) setNode(node *v1.Node) {
 	ni.setOversubscription(node)
-	ni.setNodeOthersResource(node)
 	ni.setRevocableZone(node)
 
 	ni.Name = node.Name
 	ni.Node = node
+	ni.setNodeOthersResource(node)
```

Comment:
Why move here?

Reply:
because we need ni.Name set before invoking setNodeOtherResource, otherwise, NodeName will be a nil string in metrics

Reply:
We can move `ni.Name = node.Name` to the first line.

Reply:
ok